src/task/index.html

```diff
@@ -115,7 +115,9 @@
             placeholder="eg: Marcus"
             id="assignee"
             name="assignee"
+            autocomplete="off"
           />
+          <div id="list-items"></div>
```

Comment:
Is there any easy way to get suggestions using input elements? (hint auto-complete feature)